doc/ref/modules/index.rst

```diff
@@ -405,6 +405,10 @@ similar to the following:
            return __virtualname__
        return False
 
+Note that the ``__virtual__()`` function will return either a ``True`` or ``False``
```

Comment:
They can also return tuples, typically when set to False, where the first value is a boolean and the second is an error message to display.